tests/import-name/ignored-list-fail/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+    "rules": {
+        "import-name": [
+            true,
+            {},
```

Comment:
> `{},`

This seems like a bug in `import-name`. It's looking at arguments starting at position 1 (I guess because in the unit test runner they're coming in there), but in real world usage they start at position 0.